src/main.js

```diff
@@ -82,6 +83,21 @@ let load = loader({
     },
   },
 
+  docs: {
```

Comment:
do we have something here to ensure that docs are only published for those environments which have requested publishing in their config.yml entry?

Reply:
The default is
```
    publish: process.env.NODE_ENV == 'production',
```

so this shouldn't need any special handling.

Reply:
nope, NODE_ENV is set to production even for staging apps.  We should also not be reading the NODE_ENV variable outside of `lib/main.js`

Reply:
OK, then we should pass an explicit `publish` to this argument, based on a `config.yml` value.